caas/broker.go

```diff
@@ -137,6 +138,15 @@ type Broker interface {
 	// Operator returns an Operator with current status and life details.
 	Operator(string) (*Operator, error)
 
+	// ListNamespacesByAnnotations filters namespaces by annations.
+	ListNamespacesByAnnotations(annotations k8sannotations.Annotation) ([]core.Namespace, error)
```

Comment:
@wallyworld we need to expose these `3` methods to `Broker` if we want do the controller UUID -> Name outside of the `K8sClient`.
Is it ok?